#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_tcp-manpagefix.dpatch by William Vera <billy@billy.com.mx>
##
## DP: Character cleanup for UTF-8 compatibility.

@DPATCH@
diff -urNad nemesis-1.4~/man/nemesis-tcp.1 nemesis-1.4/man/nemesis-tcp.1
--- nemesis-1.4~/man/nemesis-tcp.1	2004-10-06 20:20:56.000000000 -0500
+++ nemesis-1.4/man/nemesis-tcp.1	2008-07-02 07:08:07.000000000 -0500
@@ -9,7 +9,7 @@
 .SH NAME
 nemesis-tcp \- TCP Protocol (The Nemesis Project)
 .SH SYNOPSIS
-nemesis-tcp [-vZ?] [-a
+nemesis\-tcp [\-vZ?] [\-a
 .I ack-number
 .B ] [-d
 .I Ethernet-device
@@ -79,8 +79,8 @@
 .fi
 .in -.51
 
-within the TCP header.  Flags can be combined in the form '-fPA'.  If flags 
-are specified in the form '-f-' then the TCP header will be built without 
+within the TCP header.  Flags can be combined in the form '\-fPA'.  If flags 
+are specified in the form '\-f\-' then the TCP header will be built without 
 flags.
 .IP "-o TCP-options-file"
 This will cause nemesis-dns to use the specified
@@ -88,15 +88,15 @@
 as the options when building the TCP header for the injected packet.  TCP
 options can be up to 40 bytes in length.  The TCP options file must be created
 manually based upon the desired options.  TCP options can also be read from
-stdin by specifying '-o -' instead of a TCP-options-file.
+stdin by specifying '\-o \-' instead of a TCP-options\-file.
 .IP "-P payload-file"
 This will case nemesis-tcp to use the specified
 .I payload-file
 as the payload when injecting TCP packets.  For packets injected using the
-raw interface (where -d is not used), the maximum payload size is
+raw interface (where \-d is not used), the maximum payload size is
 65415 bytes.  For packets injected using the link layer interface
-(where -d IS used), the maximum payload size is 1380 bytes.  Payloads can also
-be read from stdin by specifying '-P -' instead of a payload file.
+(where \-d IS used), the maximum payload size is 1380 bytes.  Payloads can also
+be read from stdin by specifying '\-P \-' instead of a payload file.
 
 Windows systems are limited to a maximum payload size of 1380 bytes for TCP
 packets.
@@ -143,8 +143,8 @@
 .in -.51
 
 within the IP header.  IP fragmentation options can be specified individually 
-or combined into a single argument to the -F command line switch by separating 
-the options with commas (eg. '-FD,M') or spaces (eg. '-FM 223').  The IP 
+or combined into a single argument to the \-F command line switch by separating 
+the options with commas (eg. '\-FD,M') or spaces (eg. '\-FM 223').  The IP 
 fragmentation offset is a 13-bit field with valid values from 0 to 8189.  
 Don't fragment (DF), more fragments (MF) and the reserved flag (RESERVED or RB) 
 are 1-bit fields.  
@@ -160,7 +160,7 @@
 as the options when building the IP header for the injected packet.  IP 
 options can be up to 40 bytes in length.  The IP options file must be created
 manually based upon the desired options.  IP options can also be read from 
-stdin by specifying '-O -' instead of an IP-options-file.
+stdin by specifying '\-O \-' instead of an IP-options-file.
 .IP "-S source-IP-address"
 Specify the
 .I source-IP-address
